apps/asap-server/src/entities/research-output.ts

```diff
@@ -58,3 +63,8 @@ const parseGraphqlTeamLite = (
   id: graphqlTeam.id,
   displayName: graphqlTeam.flatData?.displayName || '',
 });
+
+const convertNotSureToUndefined = (
```

Comment:
```suggestion
const convertDecisionToBoolean = (
```
Or a name that better represents what the function does?